Resources/config/security.xml

```diff
@@ -6,7 +6,6 @@
 
     <parameters>
         <parameter key="fos_user.security.interactive_login_listener.class">FOS\UserBundle\EventListener\LastLoginListener</parameter>
-        <parameter key="fos_user.security.login_manager.class">FOS\UserBundle\Security\LoginManager</parameter>
```

Comment:
Ok so I will revert my [changes](https://github.com/FriendsOfSymfony/FOSOAuthServerBundle/pull/315/files#diff-4d3f32e7db124020203dccf0e273e276L7) on this another PR too.